src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.css

```diff
@@ -0,0 +1,308 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads-initial-setup-wizard {
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	width: 719px;
+	z-index: 9;
+	background: transparent;
+}
+
+.ads-initial-setup-show {
```

Comment:
I'd rather we actually add/remove the element from the DOM instead of using display to show/hide since this is only going to show up once